speech/system-test/quickstart.test.js

```diff
@@ -15,11 +15,17 @@
 
 'use strict';
 
-require(`../../system-test/_setup`);
-
 const path = require(`path`);
 const proxyquire = require(`proxyquire`).noPreserveCache();
+const sinon = require(`sinon`);
 const speech = proxyquire(`@google-cloud/speech`, {})();
+const test = require(`ava`);
+
+const {
```

Comment:
The tests are transpiled by Ava. Our use of async/await in our tests definitely doesn't work in Node v4.